nebula/ui/components/forms/VPNTextArea.qml

```diff
@@ -74,8 +74,10 @@ Item {
                 // After gaining initial focus or typing in TextArea the screen reader
                 // fails to narrate any accessible content and action. After regaining
                 // active focus the screen reader keeps working as expected.
-                textArea.focus = false;
-                textArea.forceActiveFocus();
+                if (Qt.platform.os === "osx") {
```

Comment:
This seems a hack... Is it a qt bug? Why this condition just for osx?

Reply:
Yes — it’s to workaround issue with accessibility we experienced for VoiceOver on macOS:
- #4556

Reply:
… which reminds me that I still had a todo on my list to file this as a bug: [QTBUG-108189](https://bugreports.qt.io/browse/QTBUG-108189).